### PR TITLE
Clean up katello installation docs

### PR DESCRIPTION
**Pull request by johnpmitsch**
Removed some of the commented out code in the install commands and
removed parts of yum commands that aren't working.

### CODE REVIEW

File: plugins/katello/nightly/installation/index.md [hunk 1]
```diff
@@ -103,14 +103,22 @@ yum-config-manager --enable rhel-7-server-extras-rpms
...
 
+Run the following command to enable the correct repos for Puppet 4
+{% highlight bash %}
+yum -y localinstall https://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm
+{% endhighlight %}
+
+<strong>OR</strong> run the following command if you want to use Puppet 3
+{% highlight bash %}
+yum -y localinstall http://yum.puppetlabs.com/puppetlabs-release-el-7.noarch.rpm
+{% endhighlight %}
```
Comment (jlsherrill) [inline plugins/katello/nightly/installation/index.md]: puppet 3 isn't even supported anymore, might as well just get rid of it.

Comment (johnpmitsch) [reply plugins/katello/nightly/installation/index.md]: updated

Comment (jlsherrill) [reply plugins/katello/nightly/installation/index.md]: still seeing a puppet 3 section?

**Comment by johnpmitsch** [comment]
@jlsherrill not sure what happened there, I pushed up the changes now

**Comment by mmoll** [comment]
Is this worth to also apply to the 3.4 docs? Especially regarding the Puppet 3 desupporting, if that's the case.

**Comment by mmoll** [comment]
what's the status here?

File: plugins/katello/nightly/installation/index.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Installation may be done manually or via our recommended approach of using [fork
 
 <div id="rhel7" markdown="1">
 {% highlight bash %}
-yum -y  --disablerepo="*" --enablerepo=rhel-7-server-rpms install yum-utils wget
+yum -y  --disablerepo="*" --enablerepo=rhel-7-server-rpms
```
Comment (evgeni) [inline plugins/katello/nightly/installation/index.md:94]: that line is noop now, as you don't pass any action to yum

Comment (johnpmitsch) [reply plugins/katello/nightly/installation/index.md:94]: @evgeni https://github.com/theforeman/theforeman.org/pull/917

**Pull request closed without merging**